src/sql/src/session/vars.rs

```diff
@@ -713,6 +716,24 @@ const PG_REPLICATION_CONNECT_TIMEOUT: ServerVar<Duration> = ServerVar {
     internal: true,
 };
 
+static DEFAULT_LOGGING_FILTER: Lazy<CloneableEnvFilter> =
+    Lazy::new(|| CloneableEnvFilter::from_str("info").expect("valid EnvFilter"));
+static LOGGING_FILTER: Lazy<ServerVar<CloneableEnvFilter>> = Lazy::new(|| ServerVar {
+    name: UncasedStr::new("mz_log_filter"),
```

Comment:
```suggestion
    name: UncasedStr::new("log_filter"),
```

We don't use the `mz_` prefix for variables (unless they conflict with some PostgreSQL variable of the same name) since the `mz`-ness is implied by them being parameters in Materialize!